code/controllers/subsystem/persistence.dm

```diff
@@ -358,4 +362,38 @@ SUBSYSTEM_DEF(persistence)
 		if(!original_human || original_human.stat == DEAD || !original_human.all_scars || original_human != ending_human)
 			original_human.save_persistent_scars(TRUE)
 		else
-			original_human.save_persistent_scars()
\ No newline at end of file
+			original_human.save_persistent_scars()
+
+/datum/controller/subsystem/persistence/proc/SaveMuck()
+	var/time_start = REALTIMEOFDAY
+	var/json_file = file("data/muck+[SSmapping.config.map_name].json")
+	fdel(json_file)
+
+	var/list/packed_muck = list()
+
+	for (var/obj/effect/decal/cleanable/C in muck)
+		if (C)
+			packed_muck.Add(list(C.pack()))
```

Comment:
blood should dry, or have a % chance to not save that tile at all (probably both) maybe a configurable "max blood tiles per map" in config so it doesn't get extremely overwhelming.

Reply:
i can do it so it lasts only for like 3 rounds or something

Reply:
done